### DISCUSSION TITLE
Add Border Programmatically

### DISCUSSION THREAD

**Discussion by Vampiro** [caplin/FlexLayout#376] [Q&A] [answered]
I see how to add tabsets, but not how I could add a border node (such as adding a left border that doesn't exist). Is this possible or do I just have to include all borders I might need in the layout json from the start?

**Comment by nealus** [comment, marked as answer]
there is no api for this, so you need to add them from the start, you can use "borderEnableAutoHide": true so they only show when tabs are added, see the 'New Features' page in the demo app

**Reply by Vampiro** [reply]
Ok, thanks for the response!
